docker/pg/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM postgres:10.14
+FROM postgres:10
```

Comment:
In the future we should upgrade the postgres version, since version 10 it is supported until November 2022

Reply:
Going to ver 12 will require some changes to the auth code. We should move for with change the current postgres abstime usage to timestamp before we move to 12+ .  We will also need to determine the best migration path for existing databases.